TROUBLESHOOTING.md

```diff
@@ -126,3 +127,45 @@ const mail = Mail.create(data);
 const body = mail.toJSON();
 console.log(body);
 ```
+
+<a name="wrapping-text"></a>
+## Wrapping Text
+
+You can write blog posts using e-mail with the help of SENDGRID API, like so:
+```javascript
+sgMail.setApiKey(process.env.SENDGRID_API_KEY);
+      let msg = {
+        to: '<your-name>@blogger.com',
+        from: '<your-name>@gmail.com',
+        subject: title,
+        html: html,
+      };
+     sgMail.send(msg);
+``` 
+You can also wrap the text in the HTML to make a multi-line blog post:
+```javascript
+<div style="white-space: pre-wrap;"> 
+<code>        int a = 10;
```

Comment:
Newline after `<code>` tag and adjust block to use two-space indentation.

Reply:
@reedsa Made the changes, thank you for reviewing this one and helping me with good code practices!